charter.md

```diff
@@ -0,0 +1,336 @@
+    - [CNCF](#cncf)
+    - [GitOps WG](#gitops-wg)
+  - [Brainstorm: Vision / Ideas](#brainstorm-vision--ideas)
+    - [Working Group Requirements for Launch](#working-group-requirements-for-launch)
+    - [Interactive Demo environment & playground](#interactive-demo-environment--playground)
+    - [Design: observe.io](#design-observeio)
+    - [Design: observe.dev](#design-observedev)
+    - [CNCF interested parties and collaboration opportunities:](#cncf-interested-parties-and-collaboration-opportunities)
+      - [SIG Instrumentation](#sig-instrumentation)
+      - [CNCF TAG App Deploy: GitOps WG](#cncf-tag-app-deploy-gitops-wg)
+  - [Personas](#personas)
+    - [Practitioner](#practitioner)
+    - [Developer](#developer)
+    - [Data Scientist / Analyst](#data-scientist--analyst)
+    - [Business Function Owner](#business-function-owner)
+  - [Technology Stack (brainstorm)](#technology-stack-brainstorm)
+    - [Architecture](#architecture)
+    - [GitOps for $allTheThings](#gitops-for-allthethings)
+    - [Platform Composition & Abstraction](#platform-composition--abstraction)
+    - [Ingress: NGINX](#ingress-nginx)
+    - [Prometheus operator](#prometheus-operator)
+    - [Service Mesh: Linkerd](#service-mesh-linkerd)
+  - [workloads: Objects of Observation (wOoO)](#workloads-objects-of-observation-wooo)
+    - [CNCF Projects: Graduated (16)](#cncf-projects-graduated-16)
+    - [Incubating (25)](#incubating-25)
+    - [CNCF Projects: Sandbox (66)](#cncf-projects-sandbox-66)
+    - [Potential example workloads](#potential-example-workloads)
+    - ["Sock Shop"](#sock-shop)
+    - [.net core example (from Microsoft)](#net-core-example-from-microsoft)
+    - [Linkerd Emojivote](#linkerd-emojivote)
+    - [podtato-head](#podtato-head)
+    - [APM Game](#apm-game)
+  - [Appendices & Resources](#appendices--resources)
+    - [Mandate (from TAG Charter)](#mandate-from-tag-charter)
+      - [Mission](#mission)
+      - [Areas considered in Scope](#areas-considered-in-scope)
+  - [Meeting Notes](#meeting-notes)
+    - [2021-12-16](#2021-12-16)
+      - [Actions](#actions)
+    - [2021-12-09](#2021-12-09)
+      - [Actions](#actions-1)
+
+## Mission Statement
+
+For cloud native workloads, observability is a key requirement, yet it’s a broad term and there are many tools and approaches available to achieve it.
+
+While the [whitepaper](https://github.com/cncf/tag-observability/blob/main/whitepaper.md) by the [CNCF TAG Observability](https://github.com/cncf/tag-observability) covers the theoretical foundations of modern observability, practitioners need more guidance on how to achieve it in real life scenarios.
+
+This working group aims to provide:
+
+- a set suite of realistic sample applications, which employ observability tools to demonstrate how practitioners can gain insight and understanding.
+- actionable, best-practice guidance on how to apply observability to real-life workloads. Observed workloads will be CNCF Projects.
```

Comment:
Maybe use "good practices" rather than "best practices"? Our space is very new and I doubt that any of us can claim to know what is best in absolute terms.

Reply:
done